tonic/transforms.py

```diff
@@ -14,22 +17,13 @@ class Compose:
         >>> ])
     """
 
-    def __init__(self, transforms):
+    def __init__(self, transforms: Callable):
         self.transforms = transforms
 
-    def __call__(self, events, sensor_size, ordering, images=None, multi_image=None):
```

Comment:
@sheiksadique this is the main change. We now receive a single argument, which we then pass on to the next transform.